sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConstants.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark
+
+// cosmos db related constants
+private object CosmosConstants {
+  val CosmosIdFieldName = "id"
+
+  object StatusCodes {
+    val ConflictStatusCode = 409
+    val ServerError = 503
```

Comment:
NIT InternalServerError

Reply:
ack

Reply:
503 is ServiceUnavailable, InternalServerError is 500 though

Reply:
Good catch Matias :-) - ultimately retry should happen on all StatusCodes >= 500

Reply:
done. we should also try on 408 and 410